proposed/http-client/http-client-meta.md

```diff
@@ -33,6 +33,16 @@ client will allow libraries to be decoupled from an implementation such as Guzzl
 specify the default behaviours.
 * The purpose is not to be opinionated about the use of middlewares (PSR-15).
 
+#### Asynchronous HTTP client
+
+The reason asynchronous support is a non-goal is because a PSR for HTTP clients 
+should not define Promises. That should be a separate PSR. Waiting for a such PSR 
```

Comment:
Thinking about a wait part: it only really matters while the PSR is WIP, but afterwards it's not an information. The fact that it should be a separate PSR and at the time of writing there was no such PSR should be enough to be mentioned IMO.